ctapipe/io/dl1writer.py

```diff
@@ -139,6 +140,16 @@ class DL1Writer(Component):
 
     overwrite = Bool(help="overwrite output file if it exists").tag(config=True)
 
+    transform_image = Bool(default_value=False).tag(config=True)
+    image_dtype = Unicode(default_value="int32").tag(config=True)
+    image_offset = Int(default_value=0).tag(config=True)
+    image_scale = Float(default_value=100.0).tag(config=True)
+
+    transform_peak_time = Bool(default_value=False).tag(config=True)
+    peak_time_dtype = Unicode(default_value="int32").tag(config=True)
+    peak_time_offset = Int(default_value=0).tag(config=True)
+    peak_time_scale = Float(default_value=10.0).tag(config=True)
```

Comment:
Set both default scales to 100?  Or even 10 for charge (0.1 p.e. seems enough), and 100 for time as we discussed elsewhere?
I don't think the offset is really needed. To use integers we just want a change of units, i.e. pure scaling with no shift. Since these are signed integers I see no need for the offset (but having it there is somewhat confusing, and could eventually be misused).

Also, the idea was to set the units such that this could be "understandable" outside ctapipe, i.e. opening a DL1 file with vitables. Is this already the case? e.g. units 0.1 pe, 0.01 ns

Reply:
There is a header keyword `SCALE` saved

Reply:
@moralejo I think you are looking at an old diff, defaults are now 100 for peak time and 10 for images.

Reply:
If nobody thinks offsets would be useful here, I can remove them.

Reply:
> Offset, Scale and Original dtype are stored in the metadata, yes

But in case there is an offset, the values one could _see_ with e.g. vitables would not just be the right quantity in certain units, right?

Reply:
vitables will only see the actual values in the table and you'd have too look into the properties for "OFFSET" and "SCALE" to make sense of them.

Reply:
@moralejo since all these metadata information (units, scales, time formats, whatever) are not hdf5 features per se but additional functionality we implement using hdf5 metadata, this is true for all conversions. vitables will show you plain mjd, the plain value, ... and you'd have to look into the metadata to see if a float was actually a time object.

Reply:
Ok, but when you hover the mouse tip over the images table you would simply read e.g. "image_UNIT: 0.1 pe". An offset would make this much less intuitive.

Reply:
The offset is just in case it becomes useful for some reason (to get asymmetric  precision for example).  It's the same thing as done in the FITS standard with the BSCALE and BZERO keywords - any FITS image can have them, and it's up to the software looking at the image to apply them or not. If you look at raw fits pixel info, you would have the same misunderstanding if you didn't look in the header.  I think it's ok to keep it in , but set the offset to 0 by default

Reply:
> The offset is just in case it becomes useful for some reason (to get an asymmetric storage range - using more bits to store positive numbers). It's the same thing as done in the FITS standard with the BSCALE and BZERO keywords - any FITS image can have them, and it's up to the software looking at the image to apply them or not. If you look at raw fits pixel info, you would have the same misunderstanding if you didn't look in the header. I think it's ok to keep it in , but set the offset to 0 by default

Ok!